server/src/server.ts

```diff
@@ -18,31 +18,29 @@ import {
 	TextDocumentChangeEvent,
 	DefinitionParams,
 	Definition,
-	Range
+	Range,
```

Comment:
Could you please revert the style changes? It's hard to review changes that related to the feature work with all the diffs.

These changes include:
- Trailing comma
- Spreading single line of code into multiple lines
- Inline typing notation
- Parentheses around anonymous function parameter list

Reply:
sorry about that! my editor was set on format on save

Reply:
I revert the `server.ts` file
but please add prettier config file for this project